.circleci/config.yml

```diff
@@ -123,6 +123,22 @@ gn-build-steps: &gn-build-steps
             (cd electron/spec && npm install)
             python electron/script/lib/dbus_mock.py ./out/Default/electron electron/spec --ci --enable-logging
           fi
+          if [ "$RUN_ARM_TESTS" == "true" ]; then
+            docker run --rm --privileged multiarch/qemu-user-static:register --reset
+            docker run -it \
+              --mount type=bind,source=/tmp/workspace,target=/tmp/workspace \
+              --rm electronbuilds/electronarm7:0.0.4 | tee version.txt
```

Comment:
This docker container needs to be different for arm64 vs arm7:
https://github.com/electron/electron/blob/3-0-x/.circleci/config.yml#L191